includes/class-wc-admin-notes.php

```diff
@@ -12,6 +12,25 @@
  * Admin Notes class.
  */
 class WC_Admin_Notes {
+	/**
+	 * Hook used for recurring "unsnooze" action.
+	 */
+	const UNSNOOZE_HOOK = 'wc_admin_unsnooze_admin_notes';
+
+	/**
+	 * Schedule events and hook appropriate actions.
+	 */
+	public static function init() {
+		$queue = WC()->queue();
+		$next  = $queue->get_next( self::UNSNOOZE_HOOK );
+
+		if ( ! $next ) {
+			$queue->schedule_recurring( time(), HOUR_IN_SECONDS, self::UNSNOOZE_HOOK );
```

Comment:
so this should only really be hit one time correct? or if someone purges out jobs manually....

Reply:
Yep. Modeled after the marketplace suggestions: https://github.com/woocommerce/woocommerce/blob/5492369fe935b94aa6a06e1acbd2a58ea0fc09c3/includes/admin/marketplace-suggestions/class-wc-marketplace-updater.php#L25-L36